articles/anomaly-detection/index.md

```diff
@@ -0,0 +1,42 @@
+# Anomaly Detection
```

Comment:
Need to add at the begining of this doc:

```
---
url: /anomaly-detection
---
```

This sets the url to just /anomaly-detection rather than /anomaly-detection/index